code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -777,7 +777,8 @@
 	M.Jitter(1)
 	..()
 
-/datum/reagent/medicine/epinephrine
+/datum/reagent/medicine/
+phrine
```

Comment:
lol...

Reply:
gimme the phrine meds NOW

Reply:
oh what the fudsjfasdifjawrjveruvjasfjaw-h5eh35qt